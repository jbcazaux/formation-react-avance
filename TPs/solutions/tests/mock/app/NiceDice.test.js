import { render } from '@testing-library/react'
import NiceDice from './NiceDice'

describe('NiceDice', () => {
  it('renders dice 1', () => {
    const { container } = render(<NiceDice value={1} />)
    expect(container).toMatchSnapshot()
  })
  it('renders dice 2', () => {
    const { container } = render(<NiceDice value={2} />)
    expect(container).toMatchSnapshot()
  })
  it('renders dice 3', () => {
    const { container } = render(<NiceDice value={3} />)
    expect(container).toMatchSnapshot()
  })
  it('renders dice 4', () => {
    const { container } = render(<NiceDice value={4} />)
    expect(container).toMatchSnapshot()
  })
  it('renders dice 5', () => {
    const { container } = render(<NiceDice value={5} />)
    expect(container).toMatchSnapshot()
  })
  it('renders dice 6', () => {
    const { container } = render(<NiceDice value={6} />)
    expect(container).toMatchSnapshot()
  })
})
