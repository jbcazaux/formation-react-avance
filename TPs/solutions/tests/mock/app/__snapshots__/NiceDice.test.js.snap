// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NiceDice renders dice 1 1`] = `
<div>
  <div
    class="dice"
  >
    <div
      class="diceColumn"
    />
    <div
      class="dot"
    />
    <div
      class="diceColumn symetrie"
    />
  </div>
</div>
`;

exports[`NiceDice renders dice 2 1`] = `
<div>
  <div
    class="dice"
  >
    <div
      class="diceColumn"
    >
      <div
        class="dot"
      />
    </div>
    <div
      class="diceColumn symetrie"
    >
      <div
        class="dot"
      />
    </div>
  </div>
</div>
`;

exports[`NiceDice renders dice 3 1`] = `
<div>
  <div
    class="dice"
  >
    <div
      class="diceColumn"
    >
      <div
        class="dot"
      />
    </div>
    <div
      class="dot"
    />
    <div
      class="diceColumn symetrie"
    >
      <div
        class="dot"
      />
    </div>
  </div>
</div>
`;

exports[`NiceDice renders dice 4 1`] = `
<div>
  <div
    class="dice"
  >
    <div
      class="diceColumn"
    >
      <div
        class="dot"
      />
      <div
        class="dot"
      />
    </div>
    <div
      class="diceColumn symetrie"
    >
      <div
        class="dot"
      />
      <div
        class="dot"
      />
    </div>
  </div>
</div>
`;

exports[`NiceDice renders dice 5 1`] = `
<div>
  <div
    class="dice"
  >
    <div
      class="diceColumn"
    >
      <div
        class="dot"
      />
      <div
        class="dot"
      />
    </div>
    <div
      class="dot"
    />
    <div
      class="diceColumn symetrie"
    >
      <div
        class="dot"
      />
      <div
        class="dot"
      />
    </div>
  </div>
</div>
`;

exports[`NiceDice renders dice 6 1`] = `
<div>
  <div
    class="dice"
  >
    <div
      class="diceColumn"
    >
      <div
        class="dot"
      />
      <div
        class="dot"
      />
      <div
        class="dot"
      />
    </div>
    <div
      class="diceColumn symetrie"
    >
      <div
        class="dot"
      />
      <div
        class="dot"
      />
      <div
        class="dot"
      />
    </div>
  </div>
</div>
`;
